frontend/src/scenes/session-recordings/player/rrweb/index.ts

```diff
@@ -68,5 +68,8 @@ export const CorsPlugin: ReplayPlugin & {
 
 export const COMMON_REPLAYER_CONFIG: Partial<playerConfig> = {
     triggerFocus: false,
-    insertStyleRules: [`.ph-no-capture {   background-image: ${PLACEHOLDER_SVG_DATA_IMAGE_URL} }`],
+    insertStyleRules: [
+        `.ph-no-capture {   background-image: ${PLACEHOLDER_SVG_DATA_IMAGE_URL} }`,
+        '@media (prefers-reduced-motion: no-preference) { .scroll-trigger:not(.scroll-trigger--offscreen).animate--slide-in { animation: var(--animation-slide-in) } }',
```

Comment:
if this fixes it then lets do it

we should have this rule in a separate variable with some explanation maybe?

```
const browserSpecificFixes = []
const shopifyThingyFix = 'blah
browserSpecificFixes.push(shopifyThingyFix)
```

etc

one fix becomes two and so on 🙈